### PR TITLE
Release v2.21.0

### PR DESCRIPTION
**Pull request by pgrzesik**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,27 @@
...
+- **Standalone:** Ensure proper resolution of runtime wrappers ([#8809](https://github.com/serverless/serverless/pull/8809)) ([1833894](https://github.com/serverless/serverless/commit/1833894856991e98e0d32701217453c413164cf3)) ([Piotr Grzesik](https://github.com/pgrzesik))
+
+### Maintenance Improvements
+
+- Custom execution role getter ([#8824](https://github.com/serverless/serverless/issues/8824)) ([12805c3](https://github.com/serverless/serverless/commit/12805c3d152d85af9dba3dd3ecfa2002a621f6a8)) ([Dmitry Shirokov](https://github.com/runk))
+- Replace `fse.exists` with `fs.promises.access` ([#8788](https://github.com/serverless/serverless/issues/8788)) ([9abe9db](https://github.com/serverless/serverless/commit/9abe9db27f26ad9d7fb55ce5fcf5bbbb9235b974)) ([Sudipto Das
+  ](https://github.com/sdas13))
+- Seclude configuration parse from internals ([#8801](https://github.com/serverless/serverless/pull/8801)) ([f274cd7](https://github.com/serverless/serverless/commit/f274cd7637e8171ee04bd174e786c7e07706343a)) ([Mariusz Nowak](https://github.com/medikoo))
```
Comment (pgrzesik) [inline CHANGELOG.md:24]: @medikoo I've "squashed" all commits from that PR into one line in Changelog - do you think that any of separate commits should still be highlighted in Changelog as separate item?

Comment (medikoo) [reply CHANGELOG.md:24]: As I checked. It's good as one item. Nothing significant that should listed separately

**Review by medikoo — APPROVED** [review]
👍

File: CHANGELOG.md (hunk 1 above)
Comment (pgrzesik) [reply CHANGELOG.md:24]: Thanks for confirming :bow:
